docs/BacwardCompatability.md

```diff
@@ -0,0 +1,51 @@
+# Compatability Requirements and Implementation Plan
+
+## What is Compatability
+Before talking about how to provide compatability, it is worthwhile to clarify what we mean by compatibility. 
+Fybrik required to provide forwards and backwards compatibility in its behaviour. Compatibility is hard, especially 
+handling issues around rollback-safety. 
+Let's try to define compatability in this way:
+- Existing clients need not be aware of a new change in order for them to continue to function as they did previously, 
+even when the change is in use.
+- Any Fybrik operations that succeeded before the new change must succeed after the change.
+- Any Fybrik operations that does not use the new change must behave the same as it did before the change.
+- Any Fybrik operations that use the new change must not cause problems (e.g. crash or degrade behavior) when issued 
+against Fybrik objects, e.g. Modules, Plotters, Blueprints and Applications created before the functionality was added.
+The objects should be smoothly promoted to the new version (if required).
+- It must be possible to round-trip Fybrik version changes (convert to different API versions and back) with no loss of 
+information (Should we support downgrade ?)
+
+For the Fybrik versioning convention see **TODO**
+
+### Backward compatibility policy
+Fybrik will support up to 2 backward minor releases, e.g. the version v1.5.x should be able to work with releases v1.4.x
+and v1.3.x
+
+## Fybrik components or features that can influence on compatability 
```

Comment:
remove "on"

Reply:
I don't think so, e.g. https://www.macmillandictionary.com/dictionary/british/influence_1#:~:text=The%20usual%20preposition%20that%20follows,a%20bad%20influence%20on%20people..

Reply:
> I don't think so, e.g. https://www.macmillandictionary.com/dictionary/british/influence_1#:~:text=The%20usual%20preposition%20that%20follows,a%20bad%20influence%20on%20people..

Wrong example. Here it is a verb, not a noun. However, we'd better ask @simanadler

Reply:
fixed